### DISCUSSION TITLE
Color for each POI category

### DISCUSSION THREAD

**Discussion by pookdeveloper** [osmandapp/OsmAnd#18455] [Ideas]
Being able to choose the color for each POI category

<table style="width:100%">
  <tr>
<th><img src="https://github.com/osmandapp/OsmAnd/assets/3384734/d00ea8ba-81b5-4a8e-be07-1efa633f47b7"  width="300" height="auto"/> </td>
  </tr>
</table>

**Comment by ThomasMeister123** [comment]
I want this too :)

**Comment by Sonwon1** [comment]
Not all of us have perfect color vision so beware of the color choices. Being able to choose the colors would be fine.
